### DISCUSSION TITLE
How to heat nozzle and bed at the same time ?

### DISCUSSION THREAD

**Discussion by ShivamJoker** [Jyers/Marlin#1387] [Q&A] [answered]
In the original firmware this was by default but in this UI even if I put the gcode command to heat both, it doesn't work on restoring power.

**Comment by Cal-LF** [comment, marked as answer]
If you choose to print from cold start, the bed heats first before the nozzle starts heating. Preheating will heat both at the same time. Gcode is not stored in memory.
